### PR TITLE
Add AccountLink event

### PR DESCRIPTION
**Pull request by suzuki-shunsuke**

### CODE REVIEW

File: linebot/event.go [hunk 1]
```diff
@@ -82,25 +83,33 @@ type Beacon struct {
...
-	Postback   *Postback
-	Beacon     *Beacon
+	ReplyToken  string
+	Type        EventType
+	Timestamp   time.Time
+	Source      *EventSource
+	Message     Message
+	Postback    *Postback
+	Beacon      *Beacon
+	AccountLink *AccountLink `json:"link"`
```
[the remark this answers is not in the document]
Comment (suzuki-shunsuke) [reply linebot/event.go]: Oh, sorry. This tag is not needed.

File: linebot/event.go [hunk 2]
```diff
@@ -82,25 +83,33 @@ type Beacon struct {
 	DeviceMessage []byte
 }
 
+// AccountLink type
+type AccountLink struct {
+	Result string
```
[the remark this answers is not in the document]
Comment (suzuki-shunsuke) [reply linebot/event.go]: Thank you for your review. I see.
I will fix them and add tests of `Event.MarshalJSON` and `Event.UnmarshalJSON`.
And I have to fix `Event.MarshalJSON` also.

**Comment by suzuki-shunsuke** [comment]
I have fixed.

File: linebot/webhook_test.go [hunk 3]
```diff
@@ -236,6 +236,19 @@ var webhookTestRequestBody = `{
                 "type":"enter",
                 "dm":"1234567890abcdef"
             }
+        },
+				{
```
Comment (sugyan) [inline linebot/webhook_test.go]: Could you please replace tabs to spaces...? (Here is just inside the JSON literal..)

File: linebot/webhook_test.go [hunk 4]
```diff
@@ -444,6 +457,19 @@ var webhookTestWantEvents = []*Event{
...
+	{
+		ReplyToken: "nHuyWiB7yP5Zw52FIkcQobQuGDXCTA",
+		Type:       EventTypeAccountLink,
+		Timestamp:  time.Date(2016, time.May, 7, 13, 57, 59, int(859*time.Millisecond), time.UTC),
+		Source: &EventSource{
+			Type:   EventSourceTypeUser,
+			UserID: "U012345678901234567890123456789ab",
+		},
+		AccountLink: &AccountLink{
+			Result: "ok",
```
Comment (sugyan) [inline linebot/webhook_test.go]: Maybe `Result: AccountLinkResultOK` is more better.

**Comment by suzuki-shunsuke** [comment]
@sugyan Thank you for your review.
I have fixed again.

**Comment by sugyan** [comment]
LGTM. Thanks! 🙆‍♂️

**Pull request merged**